src/EventStoreFactory.php

```diff
@@ -21,25 +21,25 @@ class EventStoreFactory
 {
     public function create(
         string $eventStoreName,
-        Adapter $adapter,
+        EventStore $type,
         ActionEventEmitter $actionEventEmitter,
         ContainerInterface $container,
         array $pluginServiceIds
     ): EventStore {
-        $eventStore = new EventStore($adapter, $actionEventEmitter);
+        $eventStore = new ActionEventEmitterEventStore($type, $actionEventEmitter);
```

Comment:
so ActionEventEmitterEventStore is always required? The interop config factories provided in event-store make this optional.